llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -88,6 +88,9 @@ struct AMDGPUOutgoingValueHandler : public CallLowering::OutgoingValueHandler {
                                         {MRI.getType(ExtReg)})
                         .addReg(ExtReg);
       ExtReg = ToSGPR.getReg(0);
+      if (VA.getLocVT() == MVT::i1 &&
+          MIRBuilder.getMF().getSubtarget<GCNSubtarget>().isWave64())
+        ExtReg = MIRBuilder.buildAnyExt(LLT::scalar(64), ExtReg).getReg(0);
```

Comment:
The i1 value should have been directly assigned to the register, there shouldn't be an additional, incorrect G_ANYEXT

Reply:
Done.